### ISSUE TITLE
Please fix the Sketchheads activity.

### ISSUE THREAD

**Issue by BlueStacks1884** [RemyK888/discord-together#51]
My bot can't recognize the Sketchheads activity since its the only one that doesn't work. 
and also, Discord just shut down Doodle Crew.
Code Sample:
```
client.on('messageCreate', async message => {
  if (message.content === '+gametest') {
      if(message.member.voice.channel) {
   client.discordTogether.createTogetherCode(message.member.voice.channel.id, 'sketchheads').then(async invite => {
    return message.channel.send(`${invite.code}`);
});

Terminal Error:
```
```
      throw new SyntaxError('Invalid option !');
            ^

SyntaxError: Invalid option !
```

**Comment by FreshImmuc** [comment]
#46

**Comment by Cannicide** [comment]
@BlueStacks1884 The Sketchheads activity will be available in this package once v1.3.3 releases. Sketchheads has been added to the latest version of the master branch of this repository, but the NPM package hasn't been updated to reflect these changes yet.

**Comment by RemyK888** [comment]
Hello, the project has been updated, this option is now available with version **1.3.3**
`npm i discord-together@latest`
